store/src/java/com/zimbra/cs/db/DbMailItem.java

```diff
@@ -1697,6 +1697,30 @@ public static void delete(Mailbox mbox, Collection<Integer> ids, boolean fromDum
         }
     }
 
+    /**
+     * 
+     * @param mbox
+     * @param uuid
+     * @param fromDumpster
+     * @throws ServiceException 
+     * The below delete method is used to delete files that  were shared with me
+     */
+    public void deleteSharedFile(Mailbox mbox, String uuid, boolean fromDumpster) throws ServiceException {
+        DbConnection conn = mbox.getOperationConnection();
+        PreparedStatement stmt = null;
+        try {
+            stmt = conn
+                    .prepareStatement("DELETE FROM " + getMailItemTableName(mbox, fromDumpster) + " WHERE " + "uuid=?");
+            int pos = 1;
```

Comment:
no need to declare pos here as there is only one field, can use directly 1 in the setstring call

Reply:
done